.github/workflows/rust.yml

```diff
@@ -134,3 +134,79 @@ jobs:
 
       - name: 🏃🏻‍♀️ Test
         run: CC=emcc AR=emar wasm-pack test --node --features wasm
+
+  benchmarks:
+    strategy:
+      fail-fast: false
+      matrix:
+        bits: [32, 64]
+        os:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
+        exclude:
+          - bits: 32
```

Comment:
Why are we excluding 32-bit macOS? Maybe add a comment for that too?

Reply:
There is no such thing. macOS is 64-bit only.

Reply:
oops, my bad